book_object.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_reader :rentals
+  attr_accessor :title, :book
+
+  def initialize(title, book)
+    @title = title
+    @book = book
+    @rentals = []
+  end
```

Comment:
- [ ] [REQUIRED] Good job so far 👍 I appreciate what you have done but in this `def` it will be clean if you do something like the method below 😃:

```ruby
...
def initialize(title, author)
    .
    .
    # *@rentals should be initialized. Link to the example about constructors in a previous lesson https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars
    @rentals = []
  end
  
  # *A method for adding rentals should be present. Link to the example in a previous lesson: https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship
  def add_rental(person, date)
    # *the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. 
    Rental.new(date, self, person)
  end
...
```